packages/lms-client/src/repository/ArtifactDownloadPlanner.ts

```diff
@@ -47,7 +47,7 @@ export class ArtifactDownloadPlanner {
   private readyDeferredPromise = makePromise<void>();
   private readonly logger: SimpleLogger;
   private isReadyBoolean = false;
-  private isChannelClosed: boolean = false;
+  private plannedCommited: boolean = false;
```

Comment:
for consistency with general boolean variable naming convention
```suggestion
  private isPlanCommited: boolean = false;
```